music21/test/testStream.py

```diff
@@ -284,6 +285,168 @@ def testBeamsMeasure(self):
         bMeasure.show()
 
 
+class TestMakeBeams(unittest.TestCase):
+    """Group together unittests to verify proper beams creation."""
+    @staticmethod
+    def get_beams_from_stream(stream):
+        """Helper function to return beam list for all notes and rests in the stream."""
+        return [note.beams for note in stream if isinstance(note, GeneralNote)]
```

Comment:
Should be `NotRest` instead of `GeneralNote` -- rests don't have beams
Please don't use `stream` or `note` as variable names because they can shadow imports.

Reply:
Here, I would like to keep the `GeneralNote` as I usually verify in the unit tests the beaming for all the notes to keep it clear. Rests do have the `beams` attribute which always returns an empty `Beams` object. That's fine as my unit tests are looking for that too.

Reply:
Ah!  Well, that's a MISTAKE for rests!  It shouldn't be there!  They don't have stems for sure. But okay!